test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -202,20 +176,13 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 
 		It("should validate that every linux node has a root password", func() {
 			if eng.ExpandedDefinition.Properties.IsVHDDistroForAllNodes() {
-
-				kubeConfig, err := GetConfig()
-				Expect(err).NotTo(HaveOccurred())
-				master := fmt.Sprintf("%s@%s", eng.ExpandedDefinition.Properties.LinuxProfile.AdminUsername, kubeConfig.GetServerName())
 				nodeList, err := node.GetReady()
 				Expect(err).NotTo(HaveOccurred())
-				rootPasswdCmd := fmt.Sprintf("\"sudo grep '^root:[!*]:' /etc/shadow\"")
+				rootPasswdCmd := fmt.Sprintf("\"sudo grep '^root:[!*]:' /etc/shadow\" && exit 1 || exit 0")
```

Comment:
Changed this command so it actually succeeds on a zero exit code, instead of the prior implementation which expects an error response (which made the underlying ssh foo more complicated to implement)